code/game/objects/items/weapons/storage/blackshield_kits.dm

```diff
@@ -39,6 +39,15 @@
 		new /obj/item/ammo_magazine/speed_loader_rifle_75(src)
 		new /obj/item/storage/pouch/ammo(src)
 
+/obj/item/storage/box/bs_kit/roe
+	name = "\improper Roe Primary Kit"
+	desc = "The standard Blackshield equipment kit."
+
+	populate_contents()
+		new /obj/item/gun/projectile/boltgun/light_wood(src)
+		new /obj/item/ammo_magazine/ammobox/light_rifle_257_small(src)
```

Comment:
Isn't there any speedloader you could give to the kit?

Reply:
Nope theirs no .257 speed loaders

Reply:
Then how about you code some in as well?